gapic/schema/wrappers.py

```diff
@@ -737,6 +738,27 @@ def field_headers(self) -> Sequence[str]:
 
         return next((tuple(pattern.findall(verb)) for verb in potential_verbs if verb), ())
 
+    @property
+    def http_opt(self) -> Optional[Dict[str, str]]:
+        """Return the http option for this method."""
+        http = self.options.Extensions[annotations_pb2.http].ListFields()
+        if len(http) < 1:
+            return None
+
+        http_method = http[0]
```

Comment:
Based on our offline discussion, I suggest s/`http_method`/`primary_binding`/ for clarity.